app/controllers/schools/users_controller.rb

```diff
@@ -104,7 +104,7 @@ def add_school_admin
         if existing_user&.role == 'group_admin'
           redirect = true
           notice = "As a group admin for #{existing_user.school_group.name}, this user is already able to administer " \
-                      'this school'
+                   'this school'
```

Comment:
Just realised that the message here needs translating

Reply:
added